benchmark/single-source/AngryPhonebook.swift

```diff
@@ -15,11 +15,21 @@
 import TestsUtils
 import Foundation
 
-public let AngryPhonebook = BenchmarkInfo(
-  name: "AngryPhonebook",
-  runFunction: run_AngryPhonebook,
-  tags: [.validation, .api, .String],
-  legacyFactor: 7)
+public let AngryPhonebook = [
+  BenchmarkInfo(
+    name: "AngryPhonebook",
+    runFunction: run_AngryPhonebook,
+    tags: [.validation, .api, .String],
+    legacyFactor: 7),
+  BenchmarkInfo(
```

Comment:
Minor nitpick about [naming convention](https://github.com/apple/swift/blob/master/benchmark/Naming.md) is to separate the script name by a period.

Bellow we'll define workloads for different scripts and extract the common benchmarking function so that we can do this:
````swift
  BenchmarkInfo(
    name: "AngryPhonebook.Latin",
    runFunction: { angryPhonebook($0, latin) },
    tags: t, setUpFunction: { blackHole(latin) }),
  BenchmarkInfo(
    name: "AngryPhonebook.Cyrillic",
    runFunction: { angryPhonebook($0, cyrillic) },
    tags: t, setUpFunction: { blackHole(cyrillic) }),
  BenchmarkInfo(
    name: "AngryPhonebook.Armenian",
    runFunction: { angryPhonebook($0, armenian) },
    tags: t, setUpFunction: { blackHole(armenian) })